### ISSUE TITLE
Feature Request: Ctrl-Hover to preview website in a Hover preview window

### ISSUE THREAD

**Issue by j-maffe** [Quorafind/Obsidian-Surfing#143]
### Feature Requested

I absolutely love this extension! There's one small feature that would streamline its usage: a hovering preview window. Right now, hovering while holding down ctrl does nothing, unlike the behavior over normal links with the page preview core plugin. The way I imagine it would happen is to simply prompt the hover window with a Surfing tab opened in the window. 
I can sort of do this now by creating a hover window, navigating to the note with the URL, then clicking on it. Obviously, though, this defeats the purpose of quickly looking up some information from a website.
Keep up the good work!! :D

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by Quorafind** [comment]
Yep, it should be possible to do this, but I dont have time to add this feature right now. Put it into plan soon.

**Comment by j-maffe** [comment]
Yeah that would be great. The main usage I'm thinking of is the same as how it is in Wikipedia: you hover over a link and you see the definition and summary paragraph in a hover window. It'd be perfect to have that in your note where you can link to a useful webpage instead of creating a new note specifically for it. :D

**Comment by dutry** [comment]
One year later and we still want this feature

**Comment by Quorafind** [comment]
> One year later and we still want this feature

Already added in latest version.

**Comment by dutry** [comment]
> > One year later and we still want this feature
> 
> Already added in latest version.

I am using macOS, I have turned on the Hover Popover, but it does not work fine. 

But when I have a hover window over normal links with the page preview core plugin, then in the hover window it shows the pop-up window when the mouse is on the superlink.
